### PR TITLE
[RDAPI-23557] Adding flags 'multi' and 'utf' to regex redaction

### PR DESCRIPTION
**Pull request by scaro-axway**
Adding new flags 'multi' and 'utf' to description of regex in sections "Raw Redaction" and "Trace Redaction".

## Describe the changes

New configuration flags have been added to regular expressions used by Raw redaction and Trace redaction.

## Deploy preview link

https://deploy-preview-1958--axway-open-docs.netlify.app/docs/apim_administration/apigtw_admin/admin_redactors/#redact-raw-message-content

## Checklist for contributors

Before submitting this PR, please make sure:

* [ ] You have read the [contribution guidelines](https://axway-open-docs.netlify.com/docs/contribution_guidelines/)
* [ ] You have signed the [Axway CLA](https://cla.axway.com/)
* [ ] You have verified the technical accuracy of your change
* [ ] You have verified that your change does not expose any sensitive information (passwords, keys, etc.)
* [ ] You have followed the [Markdown guidelines](https://axway-open-docs.netlify.com/docs/contribution_guidelines/writing_markdown/)  (unless this is is a Netlify CMS contribution)
* [ ] You have verified that all status checks have passed

_Put an x in the boxes that apply. This is simply a reminder of what we are going to look for before merging your change._

### CODE REVIEW

**Review by cpalmeraxway — APPROVED** [review]

File: content/en/docs/apim_administration/apigtw_admin/admin_redactors.md [hunk 1]
```diff
@@ -409,6 +409,7 @@ Supported HTML form content types are as follows:
 
 ## Redact raw message content
```
Comment (andreamussap) [inline content/en/docs/apim_administration/apigtw_admin/admin_redactors.md:411]: "The regular expression engine being used..." -> used by whom, API Gateway? 

Is this sentence really necessary. None of the other 'Redact' sections state this, they just say "You can redact JSON..", "You can redact HTML...", etc, so can't we just follow the same pattern?

Comment (andreamussap) [reply content/en/docs/apim_administration/apigtw_admin/admin_redactors.md:411]: @scaro-axway @cpalmeraxway  ??

Comment (scaro-axway) [reply content/en/docs/apim_administration/apigtw_admin/admin_redactors.md:411]: Yes, regular expression engine is a 3rd party tool used at runtime by API Gateway.
Against all the other HTTP/Form/Json/Xml parsing (that are custom implementation), regular expression is not: regular expression syntax, capabilities, limits, parsing behavior (greedy, recursion, memory usage, etc...) are due to that tool.
It looks important for user to be aware of it and know where to find documentation for it.

Comment (andreamussap) [reply content/en/docs/apim_administration/apigtw_admin/admin_redactors.md:411]: Got it! I needed to make a small editorial update. Please let me know if it's ok so I can merge this PR. Thanks.

Comment (scaro-axway) [reply content/en/docs/apim_administration/apigtw_admin/admin_redactors.md:411]: It is fine for me.

**Pull request merged**